_includes/head.html

```diff
@@ -3,6 +3,30 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+<!-- Open Graph Meta -->
+
+    <meta property="og:title" content="PySangamam - Tamil Nadu's first Python conference" />
+    <meta property="og:description" content="September 7th and 8th, 2018 IC & SR - IIT Madras " />
+    <meta property="og:url" content="http://pysangamam.org/" />
```

Comment:
@shrayasr Did you look into that HTTPS fix? If that is done, we can change this to https